src/metabase/server/handler.clj

```diff
@@ -68,7 +68,8 @@
    #'mw.ssl/redirect-to-https-middleware])
 ;; ▲▲▲ PRE-PROCESSING ▲▲▲ happens from BOTTOM-TO-TOP
 
-(defn- apply-middleware [handler]
+(defn- apply-middleware
+  [handler]
```

Comment:
I think this was a mistake but let's try not to change unrelated code, even if it's just formatting

Reply:
Oh…I absolutely change (minor) formatting issues / typos / etc. when I see them. Keep the codebase improving with things I'd never open a separate PR for.

Reply:
Good point. I still think was a mistake, as we usually don't have new lines between `defn-` and the args vector unless there's a docstring.

Reply:
According to https://guide.clojure.style/#optional-new-line-after-fn-name
both are valid style, so I'll leave it as it is. 

I changed it originally because I added a docstring, but then I removed it.